src/github.js

```diff
@@ -677,6 +677,15 @@
             });
          };
 
+         // Show repository collaborators
+         // -------
+
+         this.collaborators = function (username, cb) {
+            var command = repoPath + '/collaborators' + (username ? '/' + username : '')
```

Comment:
Once you split the method in two there  will be no need for this variable. You can use the string directly into the method call.